independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/api/QuarkusRestClientProperties.java

```diff
@@ -2,6 +2,11 @@
 
 public class QuarkusRestClientProperties {
 
+    /**
+     * Configures the maximum chunk size.
+     */
+    public static final String MAX_CHUNK_SIZE = "io.quarkus.rest.client.max-chunk-size";
```

Comment:
@geoand Is it normal we are not using our config framework for all this?

Reply:
We would need to add a new entry for sure

Reply:
Done.